### PR TITLE
Capture video with high quality.

### LINKED ISSUE #165: Video is poor quality - add option to choose video quality
### Expected behavior
Have option to choose video quality.

### Actual behavior
Video quality is poor.

### Steps to reproduce the behavior
Start WPNavigationMediaPickerViewController and take video.

##### Tested on [device], iOS [version], WPiOS [version]
iPhone 5, ios 9.

I made "hack" and fork this repository and used:
options.deliveryMode = PHVideoRequestOptionsDeliveryModeHighQualityFormat; in method - (WPMediaRequestID)videoAssetWithCompletionHandler:(WPMediaAssetBlock)completionHandler

And also in WPMediaPickerViewController.m I added imagePickerController.videoQuality = UIImagePickerControllerQualityTypeHigh; in method - (void)showMediaCaptureViewController.

### PR DESCRIPTION
**Pull request by SergioEstevao**
Fixes #165 

To test:
 * Start the picker
 * Press the +
 * Tap on the capture header
 * Select video
 * Record it
 * Add it to the Camera Roll
 * Go back to the Picker
 * Long Press/Force Touch on asset to preview it.

### CODE REVIEW

**Comment by DavidTomic** [comment]
@SergioEstevao Thank you for this fix.

I see you have extra line (options.deliveryMode):
    options.deliveryMode = PHVideoRequestOptionsDeliveryModeAutomatic;
    options.networkAccessAllowed = YES;
    options.deliveryMode = PHVideoRequestOptionsDeliveryModeHighQualityFormat;

btw. you now set video to always be HighQuality. I mean is better like this than poor, but would be even better to have option to choose quality.

File: Pod/Classes/WPPHAssetDataSource.m [hunk 1]
```diff
@@ -426,6 +426,7 @@ - (WPMediaRequestID)videoAssetWithCompletionHandler:(WPMediaAssetBlock)completio
     PHVideoRequestOptions *options = [[PHVideoRequestOptions alloc] init];
     options.deliveryMode = PHVideoRequestOptionsDeliveryModeAutomatic;
     options.networkAccessAllowed = YES;
+    options.deliveryMode = PHVideoRequestOptionsDeliveryModeHighQualityFormat;
```
Comment (astralbodies) [inline Pod/Classes/WPPHAssetDataSource.m]: As @DavidTomic mentioned you should delete line 427 above.

**Comment by SergioEstevao** [comment]
@astralbodies ready for another look.

**Comment by astralbodies** [comment]
:shipit:

**Pull request merged**